bigchaindb/commands/bigchaindb.py

```diff
@@ -95,15 +79,18 @@ def run_configure(args, skip_if_exists=False):
     if not args.yes:
         for key in ('bind', ):
             val = conf['server'][key]
-            conf['server'][key] = input_on_stderr('API Server {}? (default `{}`): '.format(key, val), val)
+            conf['server'][key] = input_on_stderr(
+                'API Server {}? (default `{}`): '.format(key, val), val)
```

Comment:
I'm wondering why those lines have been reformatted, they are <120. Was it your editor or the prehooks?

Reply:
I think it was my editor.

Reply:
just a small fyi:
pre-commits flake8 will not change code, it will only complain to you and you have to change it manually.